components.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://ui.shadcn.com/schema.json",
```

Comment:
This is needed in the starter? What's it do?

Reply:
The CLI can pull in a bunch of different things beyond just components (tailwind themes, css variables, hooks, etc.). This is not needed for what we're trying to do, but `npx shadcn add` requires this file to exist before it will pull your component :/

Reply:
I think ideally we can get things to work like:
`npx shadcn init https://reliably-numerous-kit.pgsdemo.com/components/index.json`
`npx shadcn add` which lists only our components
or `REGISTRY_URL=https://reliably-numerous-kit.pgsdemo.com/components npx shadcn add` if the above won't work.

Though when I ran init it mucked with our tailwind.config.ts and index.css. But this file was generated by init.